backend/Dockerfile

```diff
@@ -32,10 +32,10 @@ COPY backend/src/apiserver/config/sample_config.json /samples/
 # The default image is replaced with the GCR-hosted python image.
 RUN set -e; \
     < /samples/sample_config.json jq .[].file --raw-output | while read pipeline_yaml; do \
-        pipeline_py="${pipeline_yaml%.yaml}"; \
```

Comment:
Why do we change the format here?

Reply:
This is auto-formated by VS code when I am editing this file. It seems to be accurate formatting, should I revert this format change?

Reply:
I've edited the branch directly, the extra indent was intentional to make the while loop easier to read. So it seems reasonable to reserve it.

Reply:
Sounds good, I will reserve the original formatting here.